### PR TITLE
remove master.service during the non-ha to ha upgrade

### PR DESCRIPTION
**Pull request by ingvagabund**
Bug: 1506165

### CODE REVIEW

**Comment by ingvagabund** [comment]
I will spin another cluster and just verify it works for sake. We have tested the fix in the Jia's environment and it worked.

File: roles/openshift_master/tasks/clean_systemd_units.yml [hunk 1]
```diff
@@ -7,3 +7,15 @@
...
   ignore_errors: true
+
+- block:
+  - name: Remove master service
+    file:
+      path: "{{ containerized_svc_dir }}/{{ openshift.common.service_type }}-master.service"
+      state: absent
+  - name: Reset master service in systemd so it does not report as not-found
+    command: >
+      reset-failed {{ openshift.common.service_type }}-master
```
Comment (sdodson) [inline roles/openshift_master/tasks/clean_systemd_units.yml]: `systemctl reset-failed ....`

**Comment by ingvagabund** [comment]
Ok, the `systemctl reset-failed` works only under certain conditions.

**Pull request merged**